pandas/io/excel/_openpyxl.py

```diff
@@ -467,17 +467,21 @@ def write_cells(
 
 
 class _OpenpyxlReader(_BaseExcelReader):
-    def __init__(self, filepath_or_buffer: FilePathOrBuffer) -> None:
+    def __init__(
+        self, filepath_or_buffer: FilePathOrBuffer, storage_options=None
```

Comment:
can you type wiht Optional[StorageOptions]

Reply:
OK